### PR TITLE
very small changes to the docs

### PR DESCRIPTION
**Pull request by ManoloTonto1**
- reworded a few things
- added a few more examples for fine-grained reactivity

an issue was present in the docs where the "why?" section was not forked correctly from the main repo.
@jmeistrich please add the `why?.mdx` back in production.

### CODE REVIEW

**Comment by ManoloTonto1** [comment]
The first commit had issues because of my global prettier config, maybe we should consider adding a prettier config file for future  contributions

**Comment by jmeistrich** [comment]
I merged the v2 branch since you checked it out, so I rebased your branch on the merged main, and tweaked a few things.

**Comment by jmeistrich** [comment]
Would you like me to merge now or are you still working on it?

**Comment by ManoloTonto1** [comment]
> Would you like me to merge now or are you still working on it?

I think it's ready to ship but, test if everything looks good on a mock-production environment. if anything looks weird let me know.

File: docs/3-react/3-fine-grained-reactivity.mdx [hunk 1]
```diff
@@ -226,14 +226,18 @@ Props:
 </Show>
 ```
 
+<br />
+
 ```jsx
 import { Show } from '@legendapp/state/react'
 import { computed } from '@legendapp/state'
```
Comment (ManoloTonto1) [inline docs/3-react/3-fine-grained-reactivity.mdx:233]: we should maybe remove this import if we won't use computed.

I personally use the computed higher-order function when running my show components.

**Pull request merged**